### ISSUE TITLE
POST operation not treated as query despite tag override

### ISSUE THREAD

**Issue by dawsonbooth** [anymaniax/orval#730]
## What are the steps to reproduce this issue?

1. Add tag override to treat POST with "search" tag as a query

```ts
    output: {
      target: 'src/api/generated.ts',
      mode: 'single',
      client: 'react-query',
      override: {
        tags: {
          search: {
            query: {
              useQuery: true,
            },
          },
        },
      },
    },
```

## What happens?

Hooks are still generated as mutations, just moved to a different part of the file.

## What were you expecting to happen?

Hooks would be generated as queries.

## Any other comments?

#515 has a working example using `operations`. However, since we have no operation IDs on the paths, using a tag is ideal.

**Comment by mattiasnordqvist** [comment]
Not working for us either. Did you find a solution?

**Comment by dawsonbooth** [comment]
Not exactly. We're instead wrapping the mutation with a custom hook that mimics the query API, which isn't really ideal but works for now for our purposes.

**Comment by mattiasnordqvist** [comment]
How about:
`  output: {
     ...
      client: 'react-query',
      override: {
        transformer: (verbOptions) => {
          if(verbOptions.tags.includes('Query')){
            verbOptions.override.query.useQuery = true;
          }
          return verbOptions;
        },
      },
    }`

**Comment by mattiasnordqvist** [comment]
This worked for us:

  override: {
        query: {
          useQuery: false,
          useMutation: false,
        },
        transformer: (verbOptions) => {
          verbOptions.override.operations[verbOptions.operationId] = {
            ...verbOptions.override.operations[verbOptions.operationId],
            query: {
              ...verbOptions.override.operations[verbOptions.operationId]?.query,
              useMutation: verbOptions.verb == 'post' && verbOptions.tags.includes('Command'),
              useQuery: verbOptions.verb == 'get' || verbOptions.tags.includes('Query'),
            },
          };
          return verbOptions;
        },

**Comment by melloware** [comment]
Submitted a bug fix for 6.25.0
